connectors-e2e-test/connector-http-json-test/pom.xml

```diff
@@ -0,0 +1,32 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.camunda.connector</groupId>
+    <artifactId>connector-parent</artifactId>
+    <relativePath>../../parent/pom.xml</relativePath>
+    <version>8.4.0-SNAPSHOT</version>
+  </parent>
+
+  <description>Tests</description>
+  <artifactId>connector-http-json-test</artifactId>
+  <packaging>jar</packaging>
```

Comment:
Will we need this module packaged as a jar for anything apart from our CI/CD pipelines? If not, then I would suggest disabling `maven-shade-plugin` for it (originally defined in the parent) to save some build time.

Reply:
The `maven-shade-plugin` is configured for the connectors. The tests project is not inheriting from the connectors pom and therefore the plugin is not enabled.